pkg/parquetconverter/converter_marker.go

```diff
@@ -0,0 +1,61 @@
+package parquetconverter
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"path"
+
+	"github.com/cortexproject/cortex/pkg/storage/tsdb"
+	"github.com/efficientgo/core/errors"
+	"github.com/go-kit/log"
+	"github.com/oklog/ulid"
+	"github.com/thanos-io/objstore"
+	"github.com/thanos-io/thanos/pkg/runutil"
+)
+
+const (
+	ParquetConverterMakerFileName = "parquet-converter-mark.json"
+	CurrentVersion                = 1
+)
```

Comment:
Maybe consider moving those common fields to `/pkg/storage/parquet` so that can be reused by other code path like reader, bucket index, etc

Reply:
make sense!